packages/feedback-service/src/resolver.ts

```diff
@@ -1,17 +1,50 @@
 import * as _ from 'lodash';
 import { Feedback } from './schema';
 const JiraApi = require('jira-client');
+const fetch = require('node-fetch');
 import { addFeedback } from './helpers';
 
+
 export const FeedbackResolver = {
   Query: {
     listFeedback(root: any, args: any, ctx: any) {
-      return Feedback.find()
-        .populate('createdBy')
-        .sort({ 'timestamp': -1 })
-        .then(response => response)
-        .catch(err => err);
+    return Feedback.find()
+      .sort({ 'timestamp': -1 })
+      .then(response => {
+        const newresponse: any[] = [];
+        const createdBy = response.map(res => `rhatUUID_${res.createdBy}:getUsersBy(rhatUUID:"${res.createdBy}") { _id name title uid rhatUUID isActive}`);
+        const updatedBy = response.map(res => `rhatUUID_${res.updatedBy}:getUsersBy(rhatUUID:"${res.updatedBy}") { _id name title uid rhatUUID isActive}`);
+        const users = createdBy.concat(updatedBy);
+        const formattedquery = 'query' + `{${users}}`;
+        const graphql = JSON.stringify({
+          query: formattedquery,
+          variables: {}
+          });
+          const requestOptions = {
+            method: 'POST',
+            headers: { 'Content-Type': 'application/json' },
+            body: graphql
+          };
+          fetch('http://localhost:8080/graphql', requestOptions)
+          .then((result: any) => result.json())
+          .then((output: any) => {
```

Comment:
Do not use `any`. Make use of the proper types.

```suggestion
          .then(result => result.json())
          .then(output => {
```

Reply:
This is API response even i tried creating its type its not working.